### PR TITLE
Fix tests on CI

### PR DESCRIPTION
**Pull request by Nielsvanpach**
Still one tests fails on 8.0 and 8.1. I temporarily disabled it. The last part to get the CI green again, is the action running laravel-test.sh. I'm afraid I'm not really able to help with that.

### CODE REVIEW

**Comment by mr-feek** [comment]
Thanks for the pr, mind resolving conflicts?

**Comment by mr-feek** [comment]
@Nielsvanpach looks very close, but something seems up with test laravel projects that deps can't be installed?

File: tests/laravel-test.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ echo "Preparing Laravel"
 ./artisan make:test ExampleTest
 
 echo "Adding package from source"
-sed -e 's|"type": "project",|&"repositories": [ { "type": "path", "url": "../psalm-plugin-laravel" } ],|' -i composer.json
+sed -e 's|"type": "project",|&"repositories": [ { "type": "path", "url": "../psalm-plugin-laravel" } ],|' composer.json
```
Comment (mr-feek) [inline tests/laravel-test.sh]: oh, this -i is needed I'm pretty sure. why'd ya remove it, mind explaining?

Comment (Nielsvanpach) [reply tests/laravel-test.sh]: The command didn't work in my shell, but I'll restore it. Probably something Mac related.

```
sed -e "s|'type': 'project',|&'repositories': [ { 'type': 'ath', 'url': '../psalm-plugin-laravel' } ],|" -i composer.json

sed: -I or -i may not be used with stdin
```

**Comment by Nielsvanpach** [comment]
> @Nielsvanpach looks very close, but something seems up with test laravel projects that deps can't be installed?

@mr-feek Laravel 9 was used, since it's the default and this package is not yet compatible. Let's use Laravel 8 in the tests for now. Apart from that action still failing, it seems ready to be merged. Not really any changes in this PR, only running a different suite of tests so everything failing now, was already failing. It just shows up in the CI.

**Pull request merged**